src/commands/sign.ts

```diff
@@ -32,8 +32,15 @@ export default class Sign extends MuxBase {
   async run() {
     const { args, flags } = this.parse(Sign);
     const playbackId = args['playback-id'];
+    const config = await this.readConfigV1();
```

Comment:
This one I don't quite understand, because it's been a while since I wrote this. 😅 If we're signing the JWT, don't we _need_ a signing key ID and secret?

Reply:
Yea you definitely do.  I think the confusion came about because the `init` command asks if the user wants to auto-create them (https://github.com/muxinc/cli/blob/master/src/commands/init.ts#L75-L111) which it does do.  It even sets it in the `config.json` file.

But without the changes above, the command to create signing key never uses the values in the `config.json` file.  Instead, it relies on the mux-node-sdk to resolve the keys either by a parameter that is being passed in (which is what my change is) or pull it from the env vars (https://github.com/muxinc/mux-node-sdk/blob/master/src/utils/jwt.js#L20-L55).

Reply:
Oooh. Dumb on my part. Good call.